packages/@adobe/spectrum-css-temp/components/colorhandle/index.css

```diff
@@ -65,7 +65,9 @@
   }
 
   &.is-focused,
-  &.focus-ring {
+  &.focus-ring,
+  &:focus-within {
```

Comment:
I believe the color handle should only be enlarged when keyboard focused, not when dragging with the mouse.